api/public/public_api/endpoints/device_pantacor.py

```diff
@@ -0,0 +1,99 @@
+# Mycroft Server - Backend
+# Copyright (C) 2022 Mycroft AI Inc
+# SPDX-License-Identifier: 	AGPL-3.0-or-later
+#
+# This file is part of the Mycroft Server.
+#
+# The Mycroft Server is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+"""Endpoint to activate a device and finish the pairing process.
+
+A device will call this endpoint every 10 seconds to determine if the user
+has completed the device activation process on home.mycroft.ai.  The account
+API will set a Redis entry with the a key of "pairing.token" and a value of
+the pairing token generated in the pairing code endpoint.  The device passes
+the same token in the request body.  When a match is found, the activation
+is complete.
+"""
```

Comment:
Looks like this got copied across verbatim

Reply:
Look at the changes to `api.py`.  A URL for this endpoint is defined there.  The device will call this endpoint.

I will fix the copied comments